### ISSUE TITLE
Digital ocean configuration is incorrect

### ISSUE THREAD

**Issue by jaapterwoerds** [tayzlor/vagrant-puppet-mesosphere#8]
There is an error that makes it impossible to run the Vagrantfile on digital ocean.

**Comment by tayzlor** [comment]
thanks for the feedback @jaapterwoerds  - Can you paste the error or provide some more information?

**Comment by jaapterwoerds** [comment]
The error I get is
https://guides.github.com/features/mastering-markdown/:

There was an issue with the request made to the Digital Ocean
API at:
```
Path: /v2/droplets
URI Params: {}

The response status from the API was:

Status: 401
Response: {"id"=>"unauthorized", "message"=>"Unable to authenticate you."}
```

**Comment by tayzlor** [comment]
Closing. Fixed in 13fa3113a
